packages/core-data/src/hooks/test/use-entity-record.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * WordPress dependencies
+ */
+import triggerFetch from '@wordpress/api-fetch';
+import { createRegistry, RegistryProvider } from '@wordpress/data';
+
+jest.mock( '@wordpress/api-fetch' );
+
+/**
+ * External dependencies
+ */
+import TestRenderer, { act } from 'react-test-renderer';
```

Comment:
I'd recommend using `@testing-library/react` for this, but if it works it works 😂 .